README.md

```diff
@@ -14,7 +25,7 @@ For the challenge you will make code updates to `SessionBuilder.java`,
 To compile run
```

Comment:
`mvn clean package` removes any previous builds, then recompiles and creates the uber jar.

So the following is not needed, and should be removed.
```
To compile run
 javac -cp target/ModelAnswer-1.4.jar com/ibm/mq/demo/*.java
```